ykpack/src/types.rs

```diff
@@ -343,14 +344,17 @@ impl Display for Local {
     }
 }
 
-/// Bits in the `flags` bitfield in `Body`.
-pub mod bodyflags {
-    /// This function is annotated #[do_not_trace].
-    pub const DO_NOT_TRACE: u8 = 1;
-    /// This function is annotated #[interp_step].
-    pub const INTERP_STEP: u8 = 1 << 1;
-    /// This function is yktrace::trace_debug.
-    pub const TRACE_DEBUG: u8 = 1 << 2;
+bitflags! {
+    /// Bits in the `flags` bitfield in `Body`.
+    #[derive(Serialize, Deserialize)]
+    pub struct BodyFlags: u8 {
+        /// This function is annotated #[do_not_trace].
+        const DO_NOT_TRACE = 0b00000001;
```

Comment:
can we use the `1<<X` idiom? It's easier to read and less error prone.

Reply:
I used to use `1<<X` but I've made too many mistakes with it to be fond of it any more. To be honest, I didn't think deeply about this:  I think all of the examples you'll find using bitflags use the `0b` notation, which suggests that's what people have gravitated towards.